Install.php

```diff
@@ -0,0 +1,76 @@
+<?php 
+
+class PHPCompatibility_Install {
+    static function enable() {
+        echo "Enabling PHPCompatibility\n";
+        self::make_copy();
+        self::register_in_cs();
+    }
+    
+    static function disable() {
+        echo "Disabling PHPCompatibility\n";
+        self::remove_copy();
+        self::unregister_from_cs();
+    }
+    
+    static function register_in_cs() {
+        if (PHP_CodeSniffer::getConfigData('installed_paths') === NULL) {
+            PHP_CodeSniffer::setConfigData('installed_paths', __DIR__);
+            echo "Registered our path in PHP CodeSniffer\n";
+        } else if (PHP_CodeSniffer::getConfigData('installed_paths') === __DIR__ ) {
+            echo "Our path is already registered in PHP CodeSniffer\n";;
+        } else {
+            throw new Exception("Another path is registered in Code Sniffer conf :(");
```

Comment:
You can have several paths registered with PHPCS, in the `.conf` file they are a comma delimited list. PHPCS parses this to an array.
I'd need to check what the return of `getConfigData()` would be, but I think it should be possible to handle this.